vuepress/docs/v7.0/docs/README.md

```diff
@@ -5,28 +5,56 @@ sidebarDepth: 0
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means development teams can bundle and share components within a central catalog. Multiple teams can share and collaborate on a single component or whole packaged business capabilities (PBCs) across organizations and applications. They can test and assess components for readiness, manage and publish versions, and share metadata for continuous upgrades. 
+
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Bundle and Component Descriptors](../curate/ecr-bundle-details.md) | [Install Bundles from a Private Git Repository](../../tutorials/curate/ecr-private-git-repo.md) |
+| [Filtering Bundles](../curate/ecr-bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../../tutorials/curate/ecr-private-images.md) |
+| [Customize Bundle Info in App Builder](../curate/ecr-bundle-presentation-config.md) |   |
+| [Bundle Version and Updates FAQ](../curate/ecr-bundle-versions-faq.md) |  |
+| [Bundle Uninstall](../curate/ecr-uninstall-flow.md) |  |
+
+
+## Compose an Application 
+The Compose stage is centered around the [Entando App Builder](../compose/app-builder.md), a low-code composition user interface. In the App Builder, application owners can assemble new components or upgrade existing ones with templates, widgets and PBCs. The [Entando Component Manager](../compose/ecm-overview.md) and [Entando Component Repository](../compose/ecr-overview.md) are integrated into the App Builder to manage your components, making the process flexible and agile.
+
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Welcome Wizard](../compose/welcome-wizard.md) | [Page Management](../../tutorials/compose/page-management.md) |
+| [Entando Component Repository](../compose/ecr-overview.md) | [Widgets and Fragments](../../tutorials/compose/widgets-fragments.md) |
+| [Entando App Builder](../compose/app-builder.md) | [Content Creation](../../tutorials/compose/content-tutorial.md) |
+| [Entando Component Manager](../compose/ecm-overview.md) | [Content Templates](../../tutorials/compose/content-templates-tutorial.md)  |
+|  | [Digital Assets](../../tutorials/compose/digital-assets-tutorial.md) |
+
+## Consume Applications 
+In the development lifecycle of an application, Consume is the execution stage where users interact with the app. On Entando, the modular design of the components and services means they can be scaled individually, or as a whole, according to those interactions. Likewise, content can be updated, engine rules adapted, and upgrades made seamlessly. As a result, the development cycle is a continuous process with minimal disruptions and faster time to market.
```

Comment:
would remover "the" in front of "components and services" -- unnecessary and an extra word in an already full sentence

"according to those interactions" is unnecessary; it's implied, and "those" interactions makes it seem like we're talking about specific interactions instead of any interaction

"upgrades made seamlessly" reads very awkwardly

"...a continuous process with...faster time to market" doesn't make sense... the process isn't going to market --"...minimal disruptions reducing time to market"

Reply:
saying you can scale individually or as a whole isn't sufficient? 

minimal disruptions would be a contributor to faster time to market... it's just that as-is the sentence reads "a continuous process with faster time to market" - because of the "and." i'd rephrase with a verb, like "and enables" to keep the two separate